### DISCUSSION TITLE
Bug report link in Android update.

### DISCUSSION THREAD

**Discussion by sqvery** [meshtastic/Discussions#119] [Android]
In the Android client the address meshtastic.discourse.group should be updated as the site is deprecated.

Where can I check the bug reports sent via the Android client?

c-:

**Comment by rcarteraz** [comment]
> In the Android client the address meshtastic.discourse.group should be updated as the site is deprecated.

I've submitted a PR to have these updated: https://github.com/meshtastic/Meshtastic-Android/pull/1455

> Where can I check the bug reports sent via the Android client?

https://github.com/meshtastic/Meshtastic-Android/issues is for user reported issues. Anything sent via the client itself would only be visible to the developers.
